yogstation/code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -9,8 +9,23 @@
 		return
 	if(is_donator(client))
 		PickAiSkin()
```

Comment:
wouldn#t if be easier to just make the if be 
```
if(is_donator(client) && PickAiSkin())
	return
return ..()
``` ?

Reply:
I'm pretty sure the input bug was because of trying to return the parent proc. BYOND inputs are weird